src/pages/my-courses/MyCourses.tsx

```diff
@@ -32,16 +32,18 @@ const MyCourses = () => {
   const { page, handleChangePage } = usePagination()
   const { openDialog } = useConfirm()
   const { setAlert } = useSnackBarContext()
+  const [sort, setSort] = useState<string>('asc')
```

Comment:
I guess you should use useSort hook and destructure sort, and onRequestSort

Reply:
unfortunately with useSort used another format

Reply:
yeah, it will be 'updatedAt asc' and you will pass it to the backend and there just pass sort object to mongoose sort query